### ISSUE TITLE
adjust permission needed to review recommended addons

### ISSUE THREAD

**Issue by eviljeff** [mozilla/addons-server#11538]
follow up for #11072 
> Ideal: new permission called Addons:RecommendedReview, and corresponding user group.
> Otherwise: only require Addons:Review.

We may need to adjust the permissions for:
- [ ] accessing the queue
- [ ] accessing the review page
- [ ] submitting a review
- [ ] showing the links to the queue on the dashboard
- [ ] showing the tabs (jinja_helpers.py)

**Comment by diox** [comment]
@jvillalobos do we want a new group with that permission? If so, can you give me a name (`Reviewers: Recommended`?) and user ids to add to it ? Or should we add that permission to an existing group? (which one?)

**Comment by jvillalobos** [comment]
`Reviewers: Recommended` sounds like a good name. It should be empty for now (unless @wagnerand has any contributors in mind to add). The permission should also be added to the `Staff` group.

**Comment by AlexandraMoga** [comment]
I've verified the new permission on -dev and I've come up with the following results:

- the `Addons:RecommendedReview` permission is assigned to the `Reviewers: Recommended` and `Staff` groups
- as a standalone permission `Addons:RecommendedReview`  doesn't offer access to the Reviewer Tools. To make it usable, the user must also have at least:
  -  `Addons:Review` permission - which allows access to Reviewers dashboard and Pre-Review Add-ons, where the Recommended queue is located
  -  `Addons:PostReview` permission to be able to access an add-on review page and perform reviewer actions - Approve, Reject, Reply etc.

@diox is this the expected outcome?

**Comment by diox** [comment]
It's not great but not a deal breaker for now. Can you file a follow-up issue ? I've got another change coming that might fix this, we can re-check once I land it.

**Comment by AlexandraMoga** [comment]
Filed #11626 as a follow up and I'll mark this issue as verified fixed.
